public-ui/src/app/views/survey-view/survey-view.component.ts

```diff
@@ -35,6 +35,7 @@ export class SurveyViewComponent implements OnInit, OnDestroy {
   conceptCodeTooltip: any;
   binnedSurveyQuestions: string[] = ['1585864', '1585870', '1585873', '1585795', '1585802',
     '1585820', '1585889', '1585890'];
+  component = 'surveys';
```

Comment:
nit: I don't think there's much value to setting this variable, I'd probably just inline it

Reply:
Changed it to inline.